tests/test_standalone_app.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+import pytest
+import sys
+import os
+import io
+
+
+@pytest.fixture(scope="session")
+def current_path():
+    return os.path.dirname(os.path.realpath(__file__))
+
+
+@pytest.fixture(scope="session")
+def set_environment(current_path):
+    """
+    Tests the python console standalone without sgtk being present.
+    :param current_path:
+    :return:
+    """
+    # Added the path to the console app so it can be imported.
+    sys.path.insert(0, current_path + "/../python")
+
+    # make sure tk-core is removed from the path so that
+    # we can test the python console app standalone.
+    for a_path in reversed(sys.path):
```

Comment:
It's removed from the path, but how are you sure it is not already imported?

Reply:
If it's removed from the path, and I have a test making sure it can't be imported is that not enough?
If the app can't import sgtk then it shouldn't be able to use it.

Reply:
Oh, right, I missed that test, sorry!

Reply:
No worries!